src/test/resources/gradle-layout-kotlin/src/gatling/resources/search.csv

```diff
@@ -0,0 +1,3 @@
+searchCriterion,searchComputerName
+Macbook,MacBook Pro
+eee,ASUS Eee PC 1005PE
```

Comment:
EOL

Reply:
I have to check what our CSV library does on a last empty line. We don't want an empty record.